composer.json

```diff
@@ -57,10 +57,11 @@
     "symfony/yaml": "^6.4",
     "twig/twig": "^3.8",
     "web-auth/cose-lib": "^4.4",
-    "web-auth/webauthn-lib": "^5.1",
-    "web-auth/webauthn-stimulus": "^5.1",
-    "web-auth/webauthn-symfony-bundle": "^5.1",
-    "web-token/jwt-signature-algorithm-rsa": "^3.4"
+    "web-auth/webauthn-lib": "^5.2",
+    "web-auth/webauthn-stimulus": "^5.2",
+    "web-auth/webauthn-symfony-bundle": "^5.2",
+    "web-token/jwt-signature-algorithm-rsa": "^3.4",
+    "spomky-labs/pki-framework": "^1.3.0"
```

Comment:
Is the `spomky-labs/pki-framework` dependency still needed because it don't seem to be a dependency of the GSSP,
This should already be enforced by the `web-auth/cose-lib` dependency?
By removing this line it would later on be easier to upgrade the GSSP dependencies later on.